product_docs/docs/pgd/5/logical-slot-failover.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: Logical Slot Failover Support
+navTitle: Logical Slot Failover Support
+description: Logical Slot Failover Support with EDB Postgres Advanced Server and EDB Postgres Extended Server (PGD 5.7 and later only).
+---
+
+!!!warning Availability
+This is a PGD 5.7 and later feature. It is not supported on earlier versions of PGD.
+!!!
+
+## Background
+
+Earlier versions of PGD have allowed the creation of logical replication slots on nodes that can provide a feed of the logical changes happening to the data within the database. These logical replication slots have been local to the node and not replicated. Apart from only replicating changes on the particular node, this has presented challenges when faced with node failover within the cluster. In that scenario, a consumer of the logical replication off a node which fails has no replica of the slot on another node to continue consuming from.
+
+While solutions to this can be engineered using a subscriber-only node as an intermediary, it does significantly raise the cost of logical replication.
+
+## Logical Slot Failover Support
```

Comment:
I think we should also discuss the implications of consumer trying to start decoding from multiple nodes at the same time. PGD prevents that by ensuring that a Logical Failover Slot is only decoded on one node at a time. If the consumer connects to another node (or another instance of the consumer is started), then previous connection will be closed by PGD. This requires functional Raft on both the nodes though. So in case of split brain situations, PGD won't be able to guarantee that.